ODIN_II/SRC/netlist_cleanup.cpp

```diff
@@ -48,8 +49,9 @@ struct node_list_t {
 node_list_t useless_nodes;                       // List of the nodes to be removed
 node_list_t* removal_list_next = &useless_nodes; // Tail of the nodes to be removed
 
-node_list_t addsub_nodes;                      // List of the adder/subtractor nodes
-node_list_t* addsub_list_next = &addsub_nodes; // Tail of the adder/subtractor node list
+node_list_t addsub_nodes;                        // List of the adder/subtractor nodes
+node_list_t* addsub_list_next = &addsub_nodes;   // Tail of the adder/subtractor node list
```

Comment:
can you please avoid whitespace changes, it seems nothing is changed in lines 52, 53 however it shows it did

Reply:
This is result of running "make format", without running it, the CI would throw an error.